starter-code/src/app/components/log-form/log-form.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+import { AccessControlLogService } from '../../services/access-control-log.service';
+
+@Component({
+  selector: 'app-log-form',
+  templateUrl: './log-form.component.html',
+  styleUrls: ['./log-form.component.css'],
+  providers: [AccessControlLogService]
+})
+export class LogFormComponent implements OnInit {
+  constructor(private accessControlLogService: AccessControlLogService) { 
+
+  }
+
+  ngOnInit() {
+  }
+
```

Comment:
Remember to implement the method ```addAccessItem(form)``` that you are using in line 5 of log-form.component.html that IS NOT THE SAME that the method in line 12 of access-control-log.service.ts. Maybe it is better to use different names.